load_tests/load_test.py

```diff
@@ -200,7 +200,6 @@ def run_ecs_tests():
         session = get_sts_boto_session()
 
         client = session.client('ecs')
-        waiter = client.get_waiter('tasks_stopped')
         
         processes = []
 
```

Comment:
Do we till need this for loop after logic change? I don't understand what's the work for this for-loop.
At least should we remove `processes = []`  out of the for loop?

Reply:
oh you're right, thanks, processes needs to move to below.

Reply:
https://github.com/aws/aws-for-fluent-bit/pull/677

Reply:
This for loop runs the tasks, then 2nd one validates.